icekit/bin/setup-postgres.sh

```diff
@@ -10,6 +10,13 @@ EOF
 
 set -e
 
+# Is the `pv` utility available
+if [[ `pv --help > /dev/null 2>&1` ]]; then
```

Comment:
I think you don't need to use a subshell or test here. `pv` is a program and just checking its exit code should be enough:

```
if pv --help > /dev/null 2>&1; then
    ....
fi
```